hypothesis-python/scripts/basic-test.sh

```diff
@@ -38,31 +40,24 @@ if [ "$(python -c 'import sys; print(sys.version_info[:2] in ((2, 7), (3, 6)))')
   exit 0
 fi
 
-for f in tests/nocover/test_*.py; do
-  $PYTEST "$f"
-done
+$PYTEST tests/nocover/
 
 # fake-factory doesn't have a correct universal wheel
 pip install --no-binary :all: faker
 $PYTEST tests/fakefactory/
 pip uninstall -y faker
 
 if [ "$(python -c 'import platform; print(platform.python_implementation())')" != "PyPy" ]; then
-  if [ "$(python -c 'import sys; print(sys.version_info[0] == 2 or sys.version_info[:2] >= (3, 4))')" == "True" ] ; then
-    pip install .[django]
-    HYPOTHESIS_DJANGO_USETZ=TRUE python -m tests.django.manage test tests.django
-    HYPOTHESIS_DJANGO_USETZ=FALSE python -m tests.django.manage test tests.django
-    pip uninstall -y django pytz
-  fi
-
-  if [ "$(python -c 'import sys; print(sys.version_info[:2] in ((2, 7), (3, 6)))')" = "True" ] ; then
```

Comment:
I’m less clear why removing this check is okay. Do we not run these tests for other versions?

Reply:
We exit the script further up if it's not in these versions, so the condition is redundant and invites bugs - changing only *one* to 3.7 would skip these tests on Python 3 entirely!